tests/e2e/utils.go

```diff
@@ -99,6 +100,52 @@ func getActiveApplication() string {
 	return strings.TrimSpace(odoActiveApp)
 }
 
+// returns a local config value of given key and
+// returns an empty string if value is not set
+func findLocalConfigValueOfGivenKey(key string) string {
```

Comment:
this is an unnecessary long function name. Why not just `getConfigValue(key string)`?

Reply:
I like the suggested name :)

Reply:
Done